src/ddmd/backend/gflow.c

```diff
@@ -323,7 +323,7 @@ STATIC void accumrd(vec_t GEN,vec_t KILL,elem *n)
                 rdelem(&Gl,&Kl,n->E1);
                 rdelem(&Gr,&Kr,n->E2);
 
-                switch (el_returns(n->E1) * 2 | el_returns(n->E2))
+                switch (int(el_returns(n->E1) * 2) | int(el_returns(n->E2)))
```

Comment:
this is odd... so what is the type of `bool(3) * 2` if it's not already `int`?

Reply:
The cast isn't there to change the type - it's to get rid of a warning about an implicit narrowing conversion.

Reply:
I'm confused. What is the narrowing conversion that was happening without the cast?

Reply:
It turns out that in this case it was a warning about mixing `bool` and `int` in bitwise OR (the multiplcation makes the left side an `int`). I changed it so only the cast on the right is needed.